core/domain/exp_services.py

```diff
@@ -1045,11 +1046,36 @@ def compute_summary_of_exploration(exploration, contributor_id_to_add):
         exploration.tags, ratings, exp_rights.status,
         exp_rights.community_owned, exp_rights.owner_ids,
         exp_rights.editor_ids, exp_rights.viewer_ids, contributor_ids,
+        compute_exploration_contributors_summary(exploration.id),
         exploration.version, exploration_model_created_on,
         exploration_model_last_updated)
 
     return exp_summary
 
+
+def compute_exploration_contributors_summary(exploration_id):
+    """Returns a dict containing user_id as a key, and number of commits
+    made by user_id for the Exploration as a value
```

Comment:
Returns a dict whose keys are user_ids and whose values are the number of (non-revert) commits made to the given exploration by that user_id. This does not count commits which have since been reverted.